packages/server/postgres/migrations/1693991480688_truncateReflectPromptIds.ts

```diff
@@ -12,13 +11,13 @@ export async function up() {
     .insert(
       r
         .table('ReflectPrompt')
-        .filter((row) => row('id').count().gt(100))
-        .map((row) => row.merge({id: row('id').slice(0, 100)}))
+        .filter((row: RDatum) => row('id').count().gt(100))
```

Comment:
It seems bumping this version of TS Node made a few more `no implict: any` buggies appear. This (and below) popped up when I was testing a migration